core/primitives/Cargo.toml

```diff
@@ -25,6 +25,7 @@ env_logger = "0.6.0"
 log = "0.4"
 reed-solomon-erasure = "3.1.1"
 jemallocator = { version = "0.3.0", optional = true }
+sodiumoxide = "0.2.2"
```

Comment:
Hm, we already use exonum_sodiumoxide.
Actually I think we should switch to sodiumoxide now fully though, so it's fine.